specification/search/data-plane/Azure.Search/preview/2024-03-01-Preview/searchservice.json

```diff
@@ -6352,6 +6383,105 @@
       },
       "description": "The algorithm used for indexing and querying."
     },
+    "VectorSearchCompressionConfiguration": {
+      "type": "object",
+      "discriminator": "kind",
+      "properties": {
+        "name": {
+          "externalDocs": {
+            "url": "https://docs.microsoft.com/rest/api/searchservice/Naming-rules"
+          },
+          "type": "string",
+          "description": "The name to associate with this particular configuration.",
+          "x-nullable": false
+        },
+        "kind": {
+          "$ref": "#/definitions/VectorSearchCompressionKind",
+          "description": "The name of the kind of compression method being configured for use with vector search.",
+          "x-nullable": false
+        },
+        "rerankWithOriginalVectors": {
+          "type": "boolean",
+          "default": true,
+          "description": "If set to true, once the ordered set of results calculated using compressed vectors are obtained, they will be reranked again by recalculating the full-precision similarity scores. This will improve recall at the expense of latency."
+        },
+        "defaultOversampling": {
+          "type": "number",
+          "format": "double",
+          "description": "Default oversampling factor. Oversampling will internally request more documents (specified by this multiplier) in the initial search. This increases the set of results that will be reranked using recomputed similarity scores from full-precision vectors. Minimum value is 1, meaning no oversampling (1x). This parameter can only be set when rerankWithOriginalVectors is true. Higher values improve recall at the expense of latency.",
+          "x-nullable": true
+        }
+      },
+      "required": [
+        "name",
+        "kind"
+      ],
+      "description": "Contains configuration options specific to the compression method used during indexing or querying."
+    },
+    "ScalarQuantizationVectorSearchCompressionConfiguration": {
+      "x-ms-client-name": "ScalarQuantizationCompressionConfiguration",
+      "type": "object",
+      "x-ms-discriminator-value": "scalarQuantization",
+      "allOf": [
+        {
+          "$ref": "#/definitions/VectorSearchCompressionConfiguration"
+        }
+      ],
+      "properties": {
+        "sqParameters": {
+          "x-ms-client-name": "Parameters",
+          "$ref": "#/definitions/SqParameters",
+          "description": "Contains the parameters specific to Scalar Quantization."
+        }
+      },
+      "description": "Contains configuration options specific to the scalar quantization compression method used during indexing and querying."
+    },
+    "SqParameters": {
```

Comment:
Feedback from API review: Rename to ScalarQuantizationParameters

Reply:
done